--- kitty-0.27.1/setup.py~	2023-02-22 20:30:46.000000000 +0200
+++ kitty-0.27.1/setup.py	2023-02-22 20:32:42.454586120 +0200
@@ -908,6 +908,7 @@
     if not for_platform:
         update_go_generated_files(args, os.path.join(launcher_dir, appname))
     cmd = [go, 'build', '-v']
+    cmd += ['-mod=vendor']
     ld_flags = [f"-X 'kitty.VCSRevision={get_vcs_rev_define()}'"]
     if for_freeze:
         ld_flags.append("-X 'kitty.IsFrozenBuild=true'")
